### DISCUSSION TITLE
Show meta information on Vercel UI using vercel deploy CLI

### DISCUSSION THREAD

**Discussion by sergiogmuro** [vercel/community#7089] [Help]
### Summary

Hey, I want to inform an issue using vercel deploy cli and trying to post the meta information in Vercel UI. The thing is, before we are using a Vercel integrated with Git and it show on each deploy the name of the git branch, git commit and git sha
<img width="427" alt="image" src="https://github.com/vercel/community/assets/22686048/3a963935-233a-4440-a09e-d3fd2b6077b0">

But now we moved the deployment into the pipelines and we couldn't post this information using vercel CLI, 
<img width="568" alt="image" src="https://github.com/vercel/community/assets/22686048/a1b19dbf-71bc-4917-aa94-72b714c0e28c">

### parameters
we are using the next parameters
```shell
-m gitlabCommitRef="$CI_COMMIT_REF_NAME" -m gitlabCommitSha="$CI_COMMIT_SHA" -m gitlabCommitMessage="$CI_COMMIT_MESSAGE"
```

### here is our command 
```shell
vercel deploy --prebuilt --skip-domain $EXTRA_FLAGS --token=$VERCEL_TOKEN --scope $MY_SCOPE -e VERCEL_GIT_COMMIT_REF="$CI_COMMIT_REF_NAME" -m gitlabCommitRef="$CI_COMMIT_REF_NAME" -m gitlabCommitSha="$CI_COMMIT_SHA" -m gitlabCommitMessage="$CI_COMMIT_MESSAGE"
```

Could anyone help me with this?

Thanks

### Steps to Reproduce

vercel deploy --prebuilt --skip-domain $EXTRA_FLAGS --token=$VERCEL_TOKEN --scope $MY_SCOPE -e VERCEL_GIT_COMMIT_REF="$CI_COMMIT_REF_NAME" -m gitlabCommitRef="$CI_COMMIT_REF_NAME" -m gitlabCommitSha="$CI_COMMIT_SHA" -m gitlabCommitMessage="$CI_COMMIT_MESSAGE"

**Comment by nextor2k** [comment]
I am very interested to know how to make this happen as well. did you get an answer somewhere else?

[a comment by an automated account is not shown]
